misc/chrome_extension/content-script.js

```diff
@@ -0,0 +1,142 @@
+'use strict';
+
+var port = chrome.runtime.connect();
+//global since it needs to persist beyond the function scope
+var elementsToHide = [];
+
+//Parses the XPath
+//
+//sample return
+//{type:"title", value:[element, element]}
+//
+//title: The string for the type
+//xpathString: the xpath to parse
+//getTrue: BOOL, if the value return is the text instead of the element
+var parseXPath = function (title, xpathString, getText) {
```

Comment:
Maybe change `getText` for something else. `getSomething` implies a function. Change it to something like wantText, returnText or textOnly.